web/src/main/resources/application-test.properties

```diff
@@ -0,0 +1,8 @@
+spring.datasource.url=jdbc:h2:file:~/papaya_db
```

Comment:
Nie wiem, czy mi się podoba, że one tutaj są, a nie w resource'ach testów integracyjnych. Znaczy w sumie może okej, ale po pierwsze są już osobne propertiesy testowe w resource'ach testowych w `web`, a po drugie na konsoli leci error: `Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-197]` (który napotkałam już przy testach do `web`)

Reply:
Konfigurację przeniosłem do testów bo rzeczywiście tam powinna być.
Ja takiego błędu nie dostaję :thinking:

Reply:
On leci, jak się puści całego builda/testy/checka z Gradle. W przypadku tych testów w `web` @szbrnl to zauważył oryginalnie

Reply:
Jak puszczam task `test` w Gradle to nie otrzymuję tego błędu.

Reply:
Dobra, przy `test` nie wiem, czy jest. A jak nawet jak najpierw zrobisz `clean`, a potem `build`?

Reply:
Wtedy rzeczywiście mam, zaraz się temu przyjrzę.

Reply:
Dodałem `DB_CLOSE_ON_EXIT=FALSE`